developer_docs/how_to_create_a_plugin.asciidoc

```diff
@@ -2097,7 +2099,20 @@ where `::Mutations::Ducks::Delete` is your delete mutation class inheriting from
 _Requires Foreman 2.0 or higher, set requires_foreman '>= 2.0' in
 engine.rb_
 
-You can consume events from Foreman core by registering subscribers. Subscriber is a class that inherits `Foreman::BaseSubscriber`. It is recommended to store subscribers under the `/app/subscribers/my_plugin/` directory. If you have your subscriber defined register it in the plugin. Example of your engine.rb:
+You can consume events from Foreman core by registering subscribers. Subscriber is a class that inherits `Foreman::BaseSubscriber`. It is recommended to store subscribers under the `/app/subscribers/my_plugin/` directory.
```

Comment:
Does `Subscriber` here refer to what the example below calls `MySubscriber`?

Reply:
I read it as Subscriber being any class inheriting from `::Foreman::BaseSubscriber`, MySubscriber is an instance of it (in terms of an example). Perhaps a sentence before example would help `To define a Subscriber class called MySubscriber, see the following example:`